module/localisation/BallLocalisation/src/BallModel.hpp

```diff
@@ -73,7 +73,9 @@ namespace module::localisation {
 
             const Eigen::Matrix<Scalar, 3, 1> rBWw(state[PX], state[PY], field.ball_radius);
             const Eigen::Matrix<Scalar, 3, 1> rBCc_cart(Eigen::Affine3d(Hcw) * rBWw);
-            return cartesianToSpherical(rBCc_cart);
+
+            auto measurement = cartesianToSpherical(rBCc_cart);
```

Comment:
If we want to do this everywhere, maybe we should just make a utility function to convert to->from cartisian to the inverse distance version

Reply:
I mean.. could do.. but also idgaf really :stuck_out_tongue: